package.json

```diff
@@ -7,6 +7,7 @@
   },
   "dependencies": {
     "@babel/polyfill": "^7.7.0",
+    "axios": "^0.19.2",
```

Comment:
you guy was using fetch api, so axios is not necessary here

Reply:
I'm using axios at the beginning so it is installed into dependencies, but then i change my method, but did not uninstalled it.

Reply:
alright, please delete it if not using